### PR TITLE
Provide advanced features document

### PR DESCRIPTION
**Pull request by wu-sheng**
Documents provided about 5.0.0 beta new features.

### CODE REVIEW

File: docs/en/TLS.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Requirement
+Enable **direct uplink**, by following this [document](direct-uplink.md).
+
+Because of uplink through internet, with security concern, the naming mechanism didn't fit. 
+So we didn't support TLS in naming service of HTTP service.
+
+## Supported version
+5.0.0-beta +
+
+## Mutual Auth
```
Comment (hanahmily) [inline docs/en/TLS.md]: The authority is not mutual.

File: docs/en/TLS.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Only support **no mutual auth**.
+- Use this [script](../../tools/TLS/tls_key_generate.sh) if you are not familiar with how to generate key files.
+- Find `ca.srt`, and use it at client side
+- Find `server.crt` and `server.pem`. Use them at server side.
+
+## Open and config TLS
+
+### Agent config
+- Place `ca.srt` into `/ca` folder in agent package. Notice, `/ca` is not created in distribution, please create it by yourself.
```
Comment (hanahmily) [inline docs/en/TLS.md]: Typo: ca.crt, not ca.srt

**Comment by wu-sheng** [comment]
@hanahmily Typo fixed.

File: docs/en/TLS.md (hunk 2 above)
Comment (hanahmily) [reply docs/en/TLS.md]: It is not fixed

File: docs/en/TLS.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Use this [script](../../tools/TLS/tls_key_generate.sh) if you are not familiar with how to generate key files.
+- Find `ca.crt`, and use it at client side
+- Find `server.crt` and `server.pem`. Use them at server side.
+
+## Open and config TLS
+
+### Agent config
+- Place `ca.srt` into `/ca` folder in agent package. Notice, `/ca` is not created in distribution, please create it by yourself.
+
+Agent open TLS automatically after the `/ca/ca.srt` file detected.
```
Comment (hanahmily) [inline docs/en/TLS.md]: Typo

**Comment by hanahmily** [comment]
@wu-sheng Typo

**Pull request merged**